packages/devtools_app/lib/src/memory/flutter/memory_protocol.dart

```diff
@@ -28,8 +28,13 @@ class MemoryTracker {
   final List<HeapSample> samples = <HeapSample>[];
   final Map<String, List<HeapSpace>> isolateHeaps = <String, List<HeapSpace>>{};
   int heapMax;
+
+  /// Polled VM current RSS.
   int processRss;
 
+  /// Polled adb dumpsys meminfo values.
+  AdbMemoryInfo memoryInfo;
```

Comment:
nit rename to adbMemoryInfo so this is more clear

Reply:
Done.